File: Backend/RoomMe/RoomMe.API/Models/CommonCostModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class CommonCostModel
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string UserName { get; set; }
        public double Value { get; set; }
        public string Description { get; set; }
        public DateTime Date { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/common/CommonCostModel.kt
package uj.roomme.domain.common

import java.math.BigDecimal
import java.time.OffsetDateTime

data class CommonCostModel(
    val id: Int,
    val userId: Int,
    val userName: String,
    val value: BigDecimal,
    val description: String,
    val date: OffsetDateTime
)